# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/base/crashtests/crashtests.list
# Commit: a6c95ed12e51
# Full Hash: a6c95ed12e514ccf3e43f7cf11dfa317ab376543
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2024-04-11 21:36:48
# Description:
#   Bug 1890888 - Fix a crash where mCrossShadowBoundaryRange is not clear when the shadow host is removed r=smaug
#   
#   If the end point is at a shadow root, and the corresponding host node is
#   removed from DOM, we should clear this mCrossShadowBoundaryRange for it.
#   
# ==============================================================================

diff -r 86827dc0b95b -r a6c95ed12e51 dom/base/crashtests/crashtests.list
--- a/dom/base/crashtests/crashtests.list	Thu Apr 11 12:10:52 2024 -0400
+++ b/dom/base/crashtests/crashtests.list	Thu Apr 11 16:21:19 2024 +0000
@@ -275,3 +275,4 @@
 load 1887963_1.html
 load 1887963_2.html
 asserts(0-1) load 1887974.html
+load 1890888.html